projects/frontegg-app/src/lib/frontegg-auth.service.ts

```diff
@@ -293,6 +300,23 @@ export class FronteggAuthService {
     return path.startsWith(hostedLoginRedirectUrl ?? '/oauth/callback');
   }
 
+  /**
+   * @param options.maxAge optional max age
+   * @returns A subscription for step up state - true when user is stepped up, false otherwise
+   */
+  public isSteppedUp$(observer: PartialObserver<boolean>, options?: IsSteppedUpOptions): Subscription {
```

Comment:
why is it public and `stepUp` function line 318 not?

Reply:
the default is public. I added it